extensions/reactive-mssql-client/deployment/pom.xml

```diff
@@ -143,13 +140,9 @@
                                             <color>cyan</color>
                                         </log>
                                         <wait>
-                                            <!-- good docs found at: http://dmp.fabric8.io/#build-healthcheck -->
-                                            <tcp>
-                                                <mode>direct</mode>
-                                                <ports>
-                                                    <port>1433</port>
-                                                </ports>
-                                            </tcp>
+                                            <log>SQL Server is now ready for client connections</log>
+                                            <!-- Unfortunately booting this is slow, needs to set a generous timeout: -->
+                                            <time>40000</time>
```

Comment:
:+1:  thanks! Let's see if CI managed to run it now.

Reply:
There is one failure but it looks unrelated